README.md

```diff
@@ -1,45 +1,45 @@
-# JT99-Janitor
```

Comment:
I'd like to leave this readme file, edit it as needed for the new version

Reply:
Its the exact same i just deleted and reuploaded it (also idk how to undo deletes)